content/source/docs/enterprise/private/install-installer.html.md

```diff
@@ -106,6 +106,29 @@ The UI to upload these certificates looks like:
 
 ![ptfe-ca-ui](./assets/ptfe-ca-bundle.png)
 
+
+## Alternative Terraform worker image
+
+TFE runs terraform plan and apply operations in a disposable docker container. There are cases where users may require additional tools that are not available in the default docker image. To work around this limitation users can build their own image and configure TFE to use that instead. In order for this to happen the name of the alternative docker image must be set in the config by using the relevant field as shown below:
```

Comment:
```suggestion
TFE runs `terraform plan` and `terraform apply` operations in disposable Docker containers. There are cases where runs may make frequent use of additional tools that are not available in the default Docker image. To allow use of these tools for any plan or apply, users can build their own image and configure TFE to use that instead. 

In order to use an alternative Docker image, the image name must be set in the config by using the [Name] field as shown below:
```

Reply:
`[Name]` here should be replaced by the name of the field (it's important that it be in the text, not only in an image).